api/internal/conf/conf.go

```diff
@@ -43,7 +43,7 @@ var (
 	ENV              string
 	Schema           gjson.Result
 	WorkDir          = "."
-	ServerHost       = "127.0.0.1"
+	ServerHost       = ""
```

Comment:
IMO, I think we can still keep this. For someone who only wants to bind a specific IP.

Reply:
fixed.